sample/confidential_client_certificate_sample.py

```diff
@@ -12,6 +12,8 @@
     "private_key_file": "filename.pem"
         // For information about generating thumbprint and private key file, refer:
         // https://github.com/AzureAD/microsoft-authentication-library-for-python/wiki/Client-Credentials#client-credentials-with-certificate
+
+    "endpoint": "https://graph.microsoft.com/v1.0/users"
```

Comment:
for the scope, let's use `User.ReadBasic.All`

Reply:
Thanks for the suggestion! @abhidnya13 and I backport some of the relevant descriptions from the stand-alone samples to here too!